vyper/ir/compile_ir.py

```diff
@@ -755,7 +755,7 @@ def note_line_num(line_number_map, item, pos):
 
 def note_breakpoint(line_number_map, item, pos):
     # Record line number attached to pos.
-    if item == "DEBUG":
+    if item in ("BREAKPOINT", "DEBUG"):
```

Comment:
hmm

Reply:
I made this change as the breakpoints were not showing up properly in the output.

Reply:
i did not plan for the new breakpoint instruction to show up in the output. right now pc_breakpoints should be empty